docs/apps/interchain-accounts/active-channels.md

```diff
@@ -1,29 +1,33 @@
 <!--
-order: 6
+order: 8
 -->
 
 # Understanding Active Channels 
 
 The Interchain Accounts module uses [ORDERED channels](https://github.com/cosmos/ibc/tree/master/spec/core/ics-004-channel-and-packet-semantics#ordering) to maintain the order of transactions when sending packets from a controller to a host chain. A limitation when using ORDERED channels is that when a packet times out the channel will be closed. 
 
 In the case of a channel closing, a controller chain needs to be able to regain access to the interchain account registered on this channel. `Active Channels` enable this functionality. 
-When an Interchain Account is registered using `MsgRegisterInterchainAccount`, a new channel is created on a particular port. During the `OnChanOpenAck` and `OnChanOpenConfirm` steps (controller & host chain) the `Active Channel` for this interchain account
-is stored in state.
 
-It is possible to create a new channel using the same controller chain portID if the previously set `Active Channel` is now in a `CLOSED` state. This channel creation can be initialized programatically by sending a new `MsgChannelOpenInit` message like so:
+When an Interchain Account is registered using `MsgRegisterInterchainAccount`, a new channel is created on a particular port. During the `OnChanOpenAck` and `OnChanOpenConfirm` steps (on controller & host chain respectively) the `Active Channel` for this interchain account is stored in state.
+
+It is possible to create a new channel using the same controller chain portID if the previously set `Active Channel` is now in a `CLOSED` state. This channel creation can be initialized programatically by sending a new `MsgRegisterInterchainAccount` message like so:
 
 ```go
-msg := channeltypes.NewMsgChannelOpenInit(portID, string(versionBytes), channeltypes.ORDERED, []string{connectionID}, icatypes.PortID, icatypes.ModuleName)
-handler := k.msgRouter.Handler(msg)
+msg := controllertypes.NewMsgRegisterInterchainAccount(connectionID, controllerAddress, version)
```

Comment:
Hmm, I don't think this is true. This should fail if the account was created with the legacy API. I would revert these changes

Reply:
All right, thanks. So the recommended way for ibc-go >= v6 is then to use `NewMsgChannelOpenInit`, although for ibc-go < v6 is was possible to use `NewMsgRegisterInterchainAccount`, right?

Reply:
`NewMsgRegisterInterchainAccount` did not exist, so I think the recommended approach was `NewMsgChannelOpenInit` as it could be done without going through an on chain application

Reply:
>NewMsgRegisterInterchainAccount did not exist

True, I was thinking about the message to register interchain accounts that existed in the inter-tx app... Anyway, I reverted the change.